### PR TITLE
main page action panel and logo

### PR DESCRIPTION
**Pull request by GoldenJ19**

### CODE REVIEW

File: src/components/ActionPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+type Props = {}
+
+const ActionPanel = (props: Props) => {
+    return (
+        <div className='bg-blue-200 h-60 relative'>
+            <div className="action-panel-layout">
+                <div className="action-panel-button">
+                    <span className="text-sm">Next Appointment</span>
+                    <img src="next_appointment.svg" className="action-panel-image"/>
```
Comment (micahkatz) [inline src/components/ActionPanel.tsx]: There is a `react-icons` library where you can load these images from. https://react-icons.github.io/react-icons/

This one should be called `IoCalendarClearOutline`

File: src/styles/globals.css [hunk 2]
```diff
@@ -6,3 +6,23 @@
...
+  }
+
+  .action-panel-button {
+    @apply w-48 m-auto align-top text-center font-bold space-y-1;
+    height: 192px;
+  }
+
+  .action-panel-image {
+    @apply m-auto;
+    max-width:112px;
```
Comment (micahkatz) [inline src/styles/globals.css]: You should probably put max width on the container but not the images themselves. You can do this with the `max-w` class

File: src/styles/globals.css [hunk 3]
```diff
@@ -6,3 +6,23 @@
...
+    height: 112px;
```
Comment (micahkatz) [inline src/styles/globals.css]: You can also do this with the `h-...` class

File: src/styles/globals.css [hunk 4]
```diff
@@ -6,3 +6,23 @@
...
+    transform: translate(-50%, -50%);
+  }
+
+  .action-panel-button {
+    @apply w-48 m-auto align-top text-center font-bold space-y-1;
+    height: 192px;
+  }
+
+  .action-panel-image {
+    @apply m-auto;
```
Comment (micahkatz) [inline src/styles/globals.css]: Once you have finished the tailwind conversions you can move the `@apply`s out of here

File: src/styles/globals.css [hunk 5]
```diff
@@ -6,3 +6,23 @@
...
+  .action-panel-layout {
+    @apply flex m-auto absolute h-full;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  .action-panel-button {
+    @apply w-48 m-auto align-top text-center font-bold space-y-1;
+    height: 192px;
```
Comment (micahkatz) [inline src/styles/globals.css]: same here with `height`

File: src/styles/globals.css [hunk 6]
```diff
@@ -6,3 +6,23 @@
 body {
   background: var(--color-background)
 }
+
+@layer components {
+  .action-panel-layout {
+    @apply flex m-auto absolute h-full;
+    top: 50%;
```
Comment (micahkatz) [inline src/styles/globals.css]: see if there is a better way to center without using `position: absolute` and maybe use flexbox to center it. See if you can do it in tailwind

File: src/styles/globals.css [hunk 7]
```diff
@@ -6,3 +6,23 @@
 body {
   background: var(--color-background)
```
Comment (micahkatz) [inline src/styles/globals.css:7]: this is the same as `bg-background` i believe

Comment (micahkatz) [reply src/styles/globals.css:7]: for tailwind I mean

**Review by micahkatz — CHANGES_REQUESTED** [review]
In general see if you can move everything to be in tailwind

**Comment by GoldenJ19** [comment]
Made changes you suggested, and resolved conflicts. This PR is ready for merge, and is waiting on approval by you.

**Review by micahkatz — APPROVED** [review]

**Pull request merged**